### PR TITLE
Refactor the server command

### PR DESCRIPTION
**Pull request by DoctorJohn**
## Description

I reused the `importer` util added in #899 in the `server` command. This enables users to specify their schema symbols name within a module like this:

```sh
strawberry server mypackage.mymodule:myschema
```

The schema symbols name defaults to `schema` making this change backwards comaptible. (e.g. `strawberry server app` still works).

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [x] Enhancement/optimization
- [x] Documentation

## Issues Fixed or Closed by This PR

* None

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the CONTRIBUTING document.
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by botberry** [comment]
Thanks for adding the `RELEASE.md` file!



Here's a preview of the changelog:

---

This release extends the `strawberry server` command to allow the specification
of a schema symbols name within a module:

```sh
strawberry server mypackage.mymodule:myschema
```

The schema symbols name defaults to `schema` making this change backwards compatible.

File: RELEASE.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Release type: patch
+
+This release extends the `strawberry server` command to allow the specification
+of a schema symbols name within a module:
```
Comment (patrick91) [inline RELEASE.md:4]:
```suggestion
This release extends the `strawberry server` command to allow the specification
of a schema symbol name within a module:
```

File: RELEASE.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+```sh
+strawberry server mypackage.mymodule:myschema
+```
+
+The schema symbols name defaults to `schema` making this change backwards compatible.
```
Comment (patrick91) [inline RELEASE.md:10]:
```suggestion
The schema symbol name defaults to `schema` making this change backwards compatible.
```

**Review by patrick91 — APPROVED** [review]
Nice one! Only added a comment for a (potential) typo :)

**Pull request merged**